spec/integration/install_spec.cr

```diff
@@ -533,7 +533,6 @@ describe "install" do
         override = {dependencies: {
           awesome: {git: git_url(:forked_awesome), branch: "feature/super"},
         }}
-        expected_commit = git_commits(:forked_awesome).first
```

Comment:
It might be the case that this was checking that there was at least one commit.

Reply:
That seems to be the case, but the output was never used and the specs still ran without it.

Reply:
`first` will raise an exception if the array is empty, making the spec fail. I think we should keep this expectation (not in this form, but in another form)

Reply:
`git_commits(:forked_awesome).first?.should_not be_nil`